src/hooks/use-appconfig-query.ts

```diff
@@ -0,0 +1,15 @@
+import { introspect } from "../state/introspection";
+import { useQuery } from "@tanstack/react-query";
+import { AppConfig } from "../types/app-config";
+
+const getAppConfig = introspect(
+  "Get an assignment",
```

Comment:
I think this should be "Get the app config". 
Results in duplicate key warning.

Reply:
fixed